src/material-experimental/selection/selection-module.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {CommonModule} from '@angular/common';
+import {NgModule} from '@angular/core';
+import {MatTableModule} from '@angular/material/table';
+import {MatCheckboxModule} from '@angular/material/checkbox';
+import {MatSelectAll} from './select-all';
+import {MatSelection} from './selection';
+import {MatSelectionToggle} from './selection-toggle';
+import {MatSelectionColumn} from './selection-column';
+import {MatRowSelection} from './row-selection';
+
+@NgModule({
```

Comment:
We should add a TODO in here that the table-specific stuff should be in a separate module from the other selection behaviors once we move this out of experimental